Assets/SC_Inentory.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SC_Inentory : MonoBehaviour
+{
+    private bool _isFinish;
+
+    private bool salad;
+    private bool _bacon;
+    private bool CHEESE;
+
+    private void Awake()
+    {
+        GameEventSystem.OnPlayerPickUpCollectible += ASD;
+    }
+
+    private void OnDestroy()
+    {
+        GameEventSystem.OnPlayerPickUpCollectible -= ASD;
+    }
+
+    private void Update()
+    {
+        if (!(!salad && !_bacon && !CHEESE))
+        {
+            _isFinish = true;
+            Debug.Log("FINISH!");
+            SceneManager.LoadScene("Outro");
+        }
+    }
+    
+
+    public bool IsFinish
+    {
+        get => 
+            _isFinish;
+        set 
+            => _isFinish 
+                = value;
+    }
+
+    public bool Salad
+    {
+        get => salad;
+        set {
+            salad = value;
+            GameEventSystem.UpdateScore("Salad");
```

Comment:
There is no event system class in the repository. These lines will cause an error.